examples/pipelines/commoncrawl/components/image_url_deduplication/src/main.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import dask.dataframe as dd
+
+from fondant.component import DaskTransformComponent
+
+logger = logging.getLogger(__name__)
+
+
+def deduplicate_dask_dataframe(dataframe: dd.DataFrame, column: str):
+    logger.info(f"Start exact deduplication of column {column}.")
+
+    num_partitions = dataframe.npartitions
+
+    # Sort the DataFrame on the column
+    dataframe = dataframe.sort_values(column)
```

Comment:
I would calculate a `url_surtkey` for each row, and then set it as index, which will sort on it, before deduplicating.

Reply:
I think we can't set the `url_surtkey` as index. Fondant expects as index column the column `id`, if I see it correctly:
`dataframe.index = dataframe.index.rename("id")`.

Reply:
This just renames the index, it says nothing about its values. I still think a `url_surtkey` would be the most logical index.